empyrical/tests/test_stats.py

```diff
@@ -837,3 +839,17 @@ def test_tail_ratio(self, returns, expected):
             empyrical.tail_ratio(returns),
             expected,
             1)
+
+    @parameterized.expand([
+        (empty_returns, np.nan),
+        (one_return, np.nan),
+        (flat_line_1, np.nan),
+        (pos_line, 54.799662435119096),
```

Comment:
0-1. return `nan` when not enough transactions passed in
2. return `nan` when the std is zero
3. the positive line captures the correct functionality. It removes the first value, 0, and computes the mean and std of the rest of the line, then returns the value.
4. negative line should return `nan` as none of the values are positive.

Should there be additional/different tests? I'd imagine it could be improved with real data